### ISSUE TITLE
Next: highlights: images needs an "alt text" field on the content yaml

### ISSUE THREAD

**Issue by Fupete** [italia/designers.italia.it#395]
Reviewing the highlight component: the alt text of the cover images needs to be defined by the editor. It cannot be equal to the `{title}` of the highlight/section. We need a dedicated img-alt field on the content yaml, to define the alt text, or to let it empty with conscience to result in an `alt=""` (as decoratory image, if needed)

Ref: https://github.com/italia/designers.italia.it/blob/6620942fb52b20b2402cdd046930a9a6437550e6/src/components/highlight/highlight.js#L56

cc @astagi @OscarBresolin

**Comment by Fupete** [comment]
@gpeirolo a couple of questions:
1. do you have fixed it on a specified feat/branch? Maybe we can link the PR to merge the branch on next to this issue? 
2. do you have fixed it also elsewhere is an image (eg. hero, cards, ...)?

**Comment by gpeirolo** [comment]
@Fupete no need to pone other issues, I'll review the entire code base. I think most of the images (if not all of them) have an alt attribute now.

**Comment by gpeirolo** [comment]
Fixed here [edc43a1](https://github.com/italia/designers.italia.it/commit/edc43a1fafa6685f200a5cedccd901097e40da07)
